src/test/groovy/jetbrains/buildServer/auth/oauth/OAuthAuthenticationSchemeTest.groovy

```diff
@@ -136,7 +136,7 @@ class OAuthAuthenticationSchemeTest extends Specification {
             getRequestedSessionId() >> "state"
         }
         client.getAccessToken("code") >> "token"
-        client.getUserData("token") >> new OAuthUser("testUser", "Test User", "test@example.com")
+        client.getUserData("token") >> new OAuthUser("testUser", "Test User", "test@example.com", Arrays.asList("dev-"))
```

Comment:
dont assume that there is always group, default should remain as it was, add extra case for groups if required